jobs/rocoto/postanl.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
```

Comment:
I think we can just link `post.sh` as `postanl.sh` instead of duplicating a script just for the same of `postanl`.

Reply:
@aerorahul So, just a symlink beneath `jobs/rocoto`?

Reply:
Works for me, but you'll have to undo the change I asked you to make to remove `FHRGRP` and `FHRLST` from the XML for postanl.

Reply:
(sorry)

Reply:
@WalterKolczynski-NOAA Can you point me to the change you are referring to or the thread within this PR? For some reason I don't remember and/or cannot find it.

Reply:
I don't think we need `FHRGRP` at all.  The `sed` line in `post.sh` will be able to handle the `anl` case.

Reply:
Good point. FHRGRP can be removed as a variable from both the task and metatask, as it is no longer needed at all. FHRLST will need to be added back for the task to reuse the `jobs/rocoto` script.